docs/core/Graph.md

```diff
@@ -31,22 +31,23 @@ Graph is a set of graphs that share a common set of base visualization structure
+| dimension        | object   | {}                                            | Refer [Dimension](#dimension)                                                                                                                                                           |
+| locale           | object   | LOCALE.en_US                                  | Locale object for X-Axis tick values                                                                                                                                                    |
+| pan              | object   | {}                                            | Refer [Panning](../controls/Panning.md)                                                                                                                                                 |
+| padding          | object   | `{ top: 10, bottom: 5, left: 30, right: 50 }` | Refer [Padding](../controls/Padding.md)                                                                                                                                                 |
+| showHGrid        | boolean  | true                                          | Toggle to show horizontal grid                                                                                                                                                          |
+| showLabel        | boolean  | true                                          | Toggle to show axes labels                                                                                                                                                              |
+| showLegend       | boolean  | true                                          | Toggle to show graph legend                                                                                                                                                             |
+| showNoDataText   | boolean  | true                                          | Toggle to show no data text                                                                                                                                                             |
+| showShapes       | boolean  | true                                          | Toggle to show shapes in the line graph                                                                                                                                                 |
+| showVGrid        | boolean  | true                                          | Toggle to show vertical grid                                                                                                                                                            |
+| ticksCount       | number   | 3                                             | Number of ticks to show on the vertical axes, excluding the upper and lower limits, if not provided, the number of ticks will be calculated automatically depending on the Y & Y2 range |
```

Comment:
Did we just add `ticksCount` property and alphabetized the properties?

Reply:
Yes